presto-main/src/test/java/com/facebook/presto/util/TestTimeZoneUtils.java

```diff
@@ -48,6 +48,12 @@ public void test()
             DateTimeZone dateTimeZone = DateTimeZone.forID(zoneId);
             DateTimeZone indexedZone = getDateTimeZone(TimeZoneKey.getTimeZoneKey(zoneId));
 
+            // Known bug in Joda(https://github.com/JodaOrg/joda-time/issues/427)
+            // We will skip this timezone in test
+            if (!java.util.TimeZone.getTimeZone(zoneId).hasSameRules(dateTimeZone.toTimeZone())) {
```

Comment:
@dain, @haozhun, this check will only skip the incompatible 'ragoon' timezone though.